### PR TITLE
declare expvarreceiver as alpha and add to components.go

### PR DESCRIPTION
**Pull request by jamesmoessis**
**Description:** 

* Add expvar receiver to components list.
* Declare component as alpha

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9592

### CODE REVIEW

**Review by TylerHelmuth — CHANGES_REQUESTED** [review]
Please create a status header like the one detailed in this issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10116

**Comment by TylerHelmuth** [comment]
Also, if you haven't already, open a PR in the opentelemetry-collector-releases repo to add this component to the manifest so it is included in the contrib distribution.

**Comment by jamesmoessis** [comment]
Thanks @TylerHelmuth for letting me know, I wasn't aware of these extra new steps. I've pushed up the status header, and I raised a PR in the releases repo: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/149

File: receiver/expvarreceiver/README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # Expvar Receiver
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [alpha]   |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```
Comment (TylerHelmuth) [inline receiver/expvarreceiver/README.md:7]: Does the contrib link still need added?

Comment (jamesmoessis) [reply receiver/expvarreceiver/README.md:7]: ah yes, sorry, I missed that. I've added the links at the bottom.

**Review by TylerHelmuth — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**